src/desktop/add_extensions.desktop

```diff
@@ -6,6 +6,7 @@ X-SuSE-YaST-Call=scc
 X-SuSE-YaST-Argument=select_extensions
 X-SuSE-YaST-RootOnly=yes
 X-SuSE-YaST-Group=Software
+X-SuSE-YaST-Keywords=extensions,modules,software,registration,scc
```

Comment:
Maybe also `package` and `repositories` could be useful as `software` is not that much used in the Linux world IMHO...